setup.py

```diff
@@ -80,8 +77,8 @@
     "py-txi": ["py-txi"],
     "vllm": ["vllm"],
     # optional dependencies
-    "autoawq": [AUTOAWQ],
-    "auto-gptq": ["optimum", AUTOGPTQ],
+    "autoawq": [],
+    "auto-gptq": ["optimum"],
```

Comment:
We can have our own builds in our own docker images, and have the official libs/reqs here
```suggestion
    "autoawq": ["autoawq"],
    "auto-gptq": ["optimum", "auto-gptq"],
```
if libs are already installed from source, `pip install optimum-benchmark[autoawq,auto-gptq]` normally should install nothing.